scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -9,16 +9,15 @@ import "GvsUtils.wdl" as Utils
 
 workflow GvsJointVariantCalling {
     input {
-        # If `git_branch_or_tag` is not specified by a caller (i.e. integration tests), default to the current branch or
-        # tag as specified in `GvsUtils.GetToolVersions`.
-        String? git_branch_or_tag
-        # Potentially specified by a calling integration WDL.
-        String? git_hash
-
+        String call_set_identifier
         String dataset_name
+        String extract_output_gcs_dir
```

Comment:
if they pass in a directory that their PET doesn't have access to....will this fail on the very last step?